### PR TITLE
[CARBONDATA-1880] Documentation for merging small files

### PR DESCRIPTION
**Pull request by sgururajshetty**

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by ravipesala** [comment]
Can one of the admins verify this patch?

File: docs/configuration-parameters.md [hunk 1]
```diff
@@ -61,6 +61,7 @@ This section provides the details of all the configurations required for CarbonD
...
 | carbon.blockletgroup.size.in.mb | 64 MB | The data are read as a group of blocklets which are called blocklet groups. This parameter specifies the size of the blocklet group. Higher value results in better sequential IO access.The minimum value is 16MB, any value lesser than 16MB will reset to the default value (64MB). |  |
+| carbon.task.distribution | block | **block**: Setting this value will launch one task per block. This setting is suggested in case of concurrent queries and queries having big shuffling scenarios. **custom**: Setting this value will group the blocks and distribute it uniformly to the available resources in the cluster. This enhances the query performance but not suggested in case of concurrent queries and queries having big shuffling scenarios. **blocklet**: Setting this value will launch one task per blocklet. This setting is suggested in case of concurrent queries and queries having big shuffling scenarios. **merge_small_files**: Setting this value will merge all the small partitions to a size of (128 MB) during querying. The small partitions are combined to a map task to reduce the number of read task. This enhances the performance. | |
```
Comment (QiangCai) [inline docs/configuration-parameters.md]: 1. carbon.custom.block.distribution in this file is unused,  please remove it.
2. 128 MB is the default value of  spark.sql.files.maxPartitionBytes. The user can config this spark configuration.

**Pull request closed without merging**